deploy/009_verify_deployment.js

```diff
@@ -106,6 +114,42 @@ const func = async function (hre) {
         "TIMELOCK_ADMIN_ROLE should be the admin role of the EXECUTOR_ROLE"
     );
 
+    verify(
+      !(await read('HATTimelockController', {}, 'hasRole', TIMELOCK_ADMIN_ROLE, deployer)),
+      `TIMELOCK_ADMIN_ROLE should NOT be the admin role of the deployer ${deployer}`
+    );
+    // Roles granted should be the 4 + number of executors
+    // (renounced deployer role, timelock admin of itself, governance proposer and canceller roles, and executor role to the executors)
```

Comment:
Should now be 3 not 4 because deployer role is not set by default anymore with OZ 4.8.1

Reply:
well it is not let me find out why :)